addons/l10n_ma/models/res_company.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from markupsafe import Markup
+
+from odoo import api, fields, models
+
+from odoo.addons.base.models.ir_qweb_fields import nl2br
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    ice = fields.Char("ICE", help='Common Company Identifier', default='')
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    @api.model
+    def _default_company_details(self):
+        company_details = super()._default_company_details()
+        if self.env.company.country_code == 'MA':
+            company_details += Markup(nl2br(f"\nICE: {self.env.company.ice}\n"))
```

Comment:
I suppose nl2br is simply transforming `\n` into `<br>` ... so why don't you add directly `<br> `?

Reply:
Yes indeed, but Does this gonna give us any kind of advantages?
like what makes <br> better than nl2br from our POV? (Just curious)